spec/integration/outputs/index_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "spec/es_spec_helper"
+
+shared_examples "an indexer" do
+    let(:index) { 10.times.collect { rand(10).to_s }.join("") }
+    let(:type) { 10.times.collect { rand(10).to_s }.join("") }
+    let(:event_count) { 10000 + rand(500) }
+    let(:flush_size) { rand(200) + 1 }
+    let(:config) { "not implemented" }
+
+    it "ships events" do
+      insist { config } != "not implemented"
```

Comment:
same thing here, please avoid using insist as you can do this by using except very easy. :pray: :pray: :pray: